packages/dev/test-utils/src/events.ts

```diff
@@ -121,3 +121,11 @@ export function typeText(el: HTMLElement, value: string, opts?: any) {
     skipClick = true;
   }
 }
+
+// Mocks and prevents the next click's default operation
+export function mockClickDefault(opts = {}) {
+  let onClick = jest.fn().mockImplementation(e => e.preventDefault());
+  window.addEventListener('click', onClick, opts);
```

Comment:
should still default to having `once: true` i think

Reply:
ehh, I'd rather the `opts` follow the defaults of `addEventListenListener` for consistency. Perhaps the description of the util is a bit misleading by saying "next click's"